integration/single-fetch-test.ts

```diff
@@ -1464,6 +1471,67 @@ test.describe("single-fetch", () => {
     expect(await app.getHtml("#target")).toContain("Target");
   });
 
+  test("processes redirects returned outside of react router", async ({
+    page,
+  }) => {
+    let port = await getPort();
+    let cwd = await createProject({
+      "vite.config.js": await viteConfig.basic({ port }),
+      "server.mjs": EXPRESS_SERVER({
+        port,
+        customLogic: js`
+          app.use(async (req, res, next) => {
+            if (req.url === "/page.data") {
+              let { dataRedirect } = await import("react-router");
+              let response = dataRedirect("/target");
+              res.statusMessage = response.statusText;
+              res.status(response.status);
+              for (let [key, value] of response.headers.entries()) {
+                res.append(key, value);
+              }
```

Comment:
We don't export our internal logic for converting a fetch `Response` to an express `res` so for now it's assumed that the application has to do that adapter logic on their own.  I think this is ok since status/headers are all that matter?  There isn't a body on these responses